Dockerfile

```diff
@@ -1,9 +1,17 @@
+FROM node:alpine as build
+WORKDIR /build
+COPY ./ ./
+RUN npm install gulp
+RUN npm run dist
```

Comment:
what's the benefit of running a build before creating the image (as opposed to "during", like it was before)?
I've had some issues with this approach on another project.

Reply:
no matter, if it works, it works :-)

Reply:
It wasn't actually building before as the public/ folder wasn't available. If you instead ran `npm run dist` in a single stage, you'd bring unwanted files/folders in tow with the release image. By splitting it into multi-stage, you're able to copy only the items you need from the `build` stage.